11.0.Dockerfile

```diff
@@ -195,11 +182,31 @@ ONBUILD ENV ADMIN_PASSWORD="$ADMIN_PASSWORD" \
             WITHOUT_DEMO="$WITHOUT_DEMO"
 ONBUILD ARG LOCAL_CUSTOM_DIR=./custom
 ONBUILD COPY $LOCAL_CUSTOM_DIR /opt/odoo/custom
+
+# Enable setting custom uids for odoo user during build of scaffolds
+ONBUILD ARG UID=1000
+ONBUILD ARG GID=1000
+
+# Enable Odoo user and filestore
+ONBUILD RUN groupadd -g $GID odoo \
+    && useradd -md /home/odoo -s /bin/false -u $UID -g $GID odoo \
+    && mkdir -p /var/lib/odoo \
+    && chown -R odoo:odoo /var/lib/odoo \
+    && chown -R odoo:odoo /qa/artifacts \
+    && chmod a=rwX /qa/artifacts \
+    && sync
+
+# HACK Special case for Werkzeug
+ONBUILD USER odoo
+ONBUILD RUN pip install --user Werkzeug==0.14.1
```

Comment:
I can see that the latest instruction in Dockerfiles is: `ONBUILD USER odoo`

So, I think that you can just move this line to the end, after the last line, and save those other 2 `ONBUILD USER` instructions above and below.

Reply:
install moved to the end and user change removed in docker files 8.0, 11.0 and 12.0